### PR TITLE
Java bird watcher

### PR DESCRIPTION
**Pull request by ystromm**
Turns out the arrays concept exercise already contained a ported version of the csharp bird-watcher exercise.
This PR renames arrays to bird-watcher and adds markdown files for the for-loop and foreach-loop exercises.

https://github.com/exercism/v3/issues/2734

### CODE REVIEW

**Review by mikedamay — COMMENTED** [review]
I think we need to get rid of the headings in the _about.md_ files.  Is that correct @ErikSchierboom ?

**Comment by ystromm** [comment]
Thanks for the comment, I'll compare with some other tracks and see what they look like. Also I feel the text can be improved. Hope to submit a change soonish.

**Comment by ErikSchierboom** [comment]
> I think we need to get rid of the top headings in the about.md files. Is that correct @ErikSchierboom ?

That is correct!

File: languages/java/concepts/foreach-loops/about.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## For each loop
+
```
Comment (ErikSchierboom) [inline languages/java/concepts/foreach-loops/about.md:2]:
```suggestion
```

File: languages/java/concepts/for-loops/about.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+## For loop
+
```
Comment (ErikSchierboom) [inline languages/java/concepts/for-loops/about.md:2]:
```suggestion
```

File: languages/java/concepts/foreach-loops/about.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Syntax:
```
Comment (ErikSchierboom) [inline languages/java/concepts/foreach-loops/about.md:3]: Maybe add an introductory sentence like the one used for for-loops?

**Review by ErikSchierboom — APPROVED** [review]

File: languages/java/exercises/concept/bird-watcher/.meta/config.json [hunk 4]
```diff
@@ -6,8 +6,8 @@
     }
   ],
   "editor": {
-    "solution_files": ["src/main/java/BirdCount.java"],
-    "test_files": ["src/test/java/BirdCountTest.java"]
+    "solution_files": ["src/main/java/BirdWatcher.java"],
```
Comment (mirkoperillo) [inline languages/java/exercises/concept/bird-watcher/.meta/config.json:9]: ^^ you have to add yourself as exercise author

**Review by mirkoperillo — APPROVED** [review]
@ystromm LGTM thank you
You have only to add yourself as exercise author ( I leave a comment about it)

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
You happy to merge this @mirkoperillo?

**Pull request merged**